packages/dev/core/src/FlowGraph/flowGraphContext.ts

```diff
@@ -5,6 +5,7 @@ import type { FlowGraphAsyncExecutionBlock } from "./flowGraphAsyncExecutionBloc
 import type { FlowGraphBlock } from "./flowGraphBlock";
 import type { FlowGraphDataConnection } from "./flowGraphDataConnection";
 import type { FlowGraph } from "./flowGraph";
+import type { ISerializedFlowGraphContext } from "./typeDefinitions";
 import { defaultValueParseFunction, defaultValueSerializationFunction } from "./serialization";
 import type { FlowGraphCoordinator } from "./flowGraphCoordinator";
 import { Observable } from "../Misc/observable";

```

Comment:
We do use public internal variables on different areas of the framework. This specific variable is used in FlowGraphPath as well. But this could have been solved with a getter, that's true.

Reply:
If it is actually supposed to be a public internal variable, then sure. If there are public functions to deal with user variables, exposing this field seems wrong to me. This was originally also used in the glTF extension, which is why I even looked at it. :-)